spigot/src/main/java/com/convallyria/forcepack/spigot/listener/ResourcePackListener.java

```diff
@@ -79,27 +82,25 @@ public void onStatus(PlayerResourcePackStatusEvent event) {
 
     private boolean tryValidateHacks(Player player, PlayerResourcePackStatusEvent.Status status, long now) {
         final boolean tryPrevent = getConfig().getBoolean("try-to-stop-fake-accept-hacks", true);
-        if (tryPrevent) {
-            if (status == PlayerResourcePackStatusEvent.Status.ACCEPTED) {
-                if (sentAccept.containsKey(player.getUniqueId())) {
-                    plugin.log("Kicked player " + player.getName() + " because they are sending fake resource pack statuses (accepted sent twice).");
-                    ensureMainThread(() -> player.kickPlayer(Translations.DECLINED.get(player)));
-                    return true;
-                }
-                sentAccept.put(player.getUniqueId(), now);
-            } else if (status == PlayerResourcePackStatusEvent.Status.SUCCESSFULLY_LOADED) {
-                if (!sentAccept.containsKey(player.getUniqueId())) {
-                    plugin.log("Kicked player " + player.getName() + " because they are sending fake resource pack statuses (order not maintained).");
-                    ensureMainThread(() -> player.kickPlayer(Translations.DOWNLOAD_FAILED.get(player)));
-                    return true;
-                }
+        if (status == PlayerResourcePackStatusEvent.Status.ACCEPTED) {
+            if (sentAccept.containsKey(player.getUniqueId()) && tryPrevent) {
```

Comment:
```java
if (tryPrevent && sentAccept...
```
We should have tryPrevent as the first statement in the if for a minor performance gain.

Do this for the others too